globalinfotek/linux-dev-practice
# Lide 003

_**WARNING:** Do *NOT* merge this branch until LIDE-002 has been merged._

DESCRIPTION:

Added Tedious Start (TEST) wheel to the repo `lib` directory
Added Ansible task to install TEST
Added new "Install TEST" task to the Ansible playbook

CODE REVIEW:

1. Update the CDE

    git checkout LIDE-002
    git pull
    Choose one of [these](https://github.com/globalinfotek/linux-dev-practice/wiki/CDE-Update) update options

2. Verify results

Did you see this (or something similar) in the Vagrant output?

```
    default: TASK [Install Tedious Start (TEST)] ********************************************
    default: changed: [127.0.0.1]
    default: 
```
Do you see the TEST help when you execute this command in the CDE?

```
python3 -c "import tediousstart; help(tediousstart)"
```

If the answer to all these questions is yes, then merge.
